swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
+import uuid
+
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        if id is None:
+            self.id = uuid.uuid4().int
+        else:
+            self.id = id
+
+        self.condition = condition
+
+    def get_category(self):
+        return "Item"
+
+    def __str__(self):
+        return f"An object of type Item with id {self.id}."
+
+    def condition_description(self):
+        condition = ["0/5", "1/5", "2/5", "3/5", "4/5", "5/5"]
```

Comment:
Mapping `0` through `5` to elements in a list is pretty clever! Some languages like Lua actually force you to think of lists this way. One thing to think about here: nothin constrains users of your class to a specific range for `self.condition`. How might you modify this to handle a condition such as `2.4`?